src/archivist/services/index.js

```diff
@@ -19,7 +19,14 @@ export async function load() {
   const serviceFileNames = fileNames.filter(fileName => path.extname(fileName) == '.json' && !fileName.includes('.history.json'));
 
   await Promise.all(serviceFileNames.map(async fileName => {
-    const serviceDeclaration = JSON.parse(await fs.readFile(path.join(declarationsPath, fileName)));
+    const jsonDeclarationFilePath = path.join(declarationsPath, fileName);
+    let serviceDeclaration;
+
+    try {
+      serviceDeclaration = JSON.parse(await fs.readFile(jsonDeclarationFilePath));
+    } catch (e) {
+      throw new Error(`JSON for document ${jsonDeclarationFilePath} could not be parsed`);
```

Comment:
I suggest:
`The "${path.basename(fileName, '.json')}" service declaration is malformed and cannot be parsed in ${jsonDeclarationFilePath}`

Example:
`The "Facebook" service declaration is malformed and cannot be parsed in /Users/ndpnt/Workspace/OTA/services-all/declarations/Facebook.json`

Reply:
I changed even though I believe reusing `path.basename(fileName, '.json')` is overkill